crates/biome_css_parser/src/syntax/selector.rs

```diff
@@ -364,9 +365,15 @@ pub(crate) fn parse_pseudo_element_function(p: &mut CssParser) -> ParsedSyntax {
     parse_selector(p).or_add_diagnostic(p, expect_any_selector);
 
     let context = selector_lex_context(p);
-    p.expect_with_context(T![')'], context);
+    if !p.eat_with_context(T![')'], context)
+        && ParseRecovery::new(CSS_BOGUS, PSEUDO_ELEMENT_FUNCTION_RECOVERY_SET)
```

Comment:
Any chance to have a better bogus here? Usually, we try to give specific names based on what we are parsing. This means that eventually `CSS_BOGUS` will disappear. I initially created it because we didn't have a parser yet.

Reply:
I'm not sure, because it could be anything. We may need to skip some nodes while searching for '{'.